sources/packages/backend/libs/sims-db/src/entities/notification-metadata.type.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Metadata for notification.
+ */
+export interface NotificationMetadata {
+  disbursementId: number;
```

Comment:
Since `NotificationMetadata` is supposed to map any future metadata, `disbursementId` would probably needed to nullable.
Not a blocker right now I would recommend changing it.